docs/docsite/rst/guide_networking.rst

```diff
@@ -0,0 +1,11 @@
+*****************
+Networking Guides
+*****************
+
+This section explores particular Ansible Networking functionality and use cases in greater depth and provide a more "top down" explanation of some basic features.
```

Comment:
"provide"->"provides". Also, I'd consider changing "top down explanation" to "an overview of some basic features". We're trying to minimize colloquialisms in the docs that might make future localization work more difficult.